scripts-circleci/github-package-helm-dispatch.sh

```diff
@@ -4,7 +4,7 @@ VERSION_FILE=${VERSION_FILE:="/workspace/version.txt"}
 GITHUB_WORKFLOW=${GITHUB_WORKFLOW:="package-helm.yaml"}
 
 if [ -z "${CIRCLE_TAG}" ]; then
-  if [ -z "${CIRCLE_BRANCH##*release*}" ]; then
+  if [ -z "${CIRCLE_BRANCH##*released*}" ]; then
```

Comment:
Branches with a name starting with "released/" or "to-be-released/" are handled by QR. Thus, events for such branches, like "creation of a PR from to-be-released/<some_branch> to released/<some_branch>" or "push to released/<some_branch>" should not trigger any builds or workflows (except qr_check.yaml and qr_release.yaml, which are handled by QR).